src/main/java/com/minelittlepony/model/gear/SaddleBags.java

```diff
@@ -60,7 +60,7 @@ public void init(float yOffset, float stretch) {
         x += 3;
 
         rightBag.offset(-x, y, z).around(0, 4, 4)
-                .tex(56, 25).south(0, 0, 0, 3, 6, stretch)
+                .tex(56, 25).flip().south(0, 0, 0, 3, 6, stretch)
```

Comment:
Can you align these number? That's gonna trigger my OCD

Reply:
What would be the best way to do that? Add a bunch of spaces to the line below between `.tex()` and `.south()`?

Reply:
You can decide for yourself. In general the method calls and parameters are aligned to a grid.

Reply:
Moved `flip()` call to another line in last commit.